app/models/species.rb

```diff
@@ -1,4 +1,4 @@
-class Specie < ApplicationRecord
+class Species < ApplicationRecord
```

Comment:
In general, model names should be singular, any reason why you're using a plural subject?